app/viewmodels/SummaryCardHelper.scala

```diff
@@ -403,6 +402,7 @@ class SummaryCardHelper @Inject()(link: views.html.components.link) extends Impl
   }
 
   def showDueOrPartiallyPaidDueTag(financial: Option[Financial])(implicit messages: Messages): Tag = financial match {
+    case Some(financial) if (financial.outstandingAmountDue == 0) => renderTag(messages("status.paid"))
     case Some(financial) if (financial.outstandingAmountDue < financial.amountDue && financial.outstandingAmountDue > 0) =>
```

Comment:
Do we need to check '&& financial.outstandingAmountDue > 0' looks like we are doing the check in the first case statement  'if (financial.outstandingAmountDue == 0)' . Please correct me if I'm wrong.

Reply:
You are correct, the '&&  financial.outstandingAmountDue > 0' check isn't needed anymore. Will edit and push up the change.